pallets/farming/src/benchmarking.rs

```diff
@@ -160,6 +160,14 @@ benchmarks! {
     }: {
         Pallet::<T>::vest_account_rewards(accounts);
     }
+
+    set_lp_min_xor_for_bonus_reward{
+        let root = RawOrigin::Root;
+        let balance = balance!(10000000);
+    }: {
+       let _ = Pallet::<T>::set_lp_min_xor_for_bonus_reward(root.into(), balance);
```

Comment:
```suggestion
       Pallet::<T>::set_lp_min_xor_for_bonus_reward(root.into(), balance).unwrap();
```

Reply:
It allows us to get panic if the code returns the error. Just `let _ = ...` is ok with the error result.